customize.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  version: "1.2.0"
```

Comment:
This should be done in the master branch

Reply:
revertido